hack/e2e.sh

```diff
@@ -78,20 +78,42 @@ function append_api_group() {
   echo "${resource}.ipam.metal3.io"
 }
 
+function kubectl_get_jsonpath() {
+  local OBJECT_KIND="${1}"
+  local OBJECT_NAME="${2}"
+  local JSON_PATH="${3}"
+  local n=0
+  until [ $n -ge 5 ]; do
```

Comment:
Should we use more retries. Maybe 10 or 30? Just to avoid another itreation because 5s is sometimes not enough

Reply:
I'll bump to 30 retries.

Normally it should be pretty fast.